custom_metric/README.md

```diff
@@ -0,0 +1,33 @@
+# Custom Metric
+
+1. Run the server with `temporal server start-dev`
```

Comment:
Have the sample refer to the top-level README for prerequisites like Python version, starting local server, etc. We try to keep this in a common place.